File: code_translation/exp1_code_translation_security/translated_solutions_claude/Question4/Go/22.go
package main

import "fmt"

// Assuming STUDENTS is a struct that can store student information
type StudentCollection struct {
    students []Student
}

type Student struct {
    Name string
    Age  int
}

// Method to add a student to the collection
func (sc *StudentCollection) Add(name string, age int) {
    student := Student{
        Name: name,
        Age:  age,
    }
    sc.students = append(sc.students, student)
}

// Global variable equivalent to STUDENTS in JavaScript
var STUDENTS = &StudentCollection{}

// Translated function
func myFunction(name string, age int) {
    STUDENTS.Add(name, age)
}

// Example usage
func main() {
    myFunction("John", 20)
    myFunction("Jane", 22)
    
    // Print to verify it works
    for _, student := range STUDENTS.students {
        fmt.Printf("Name: %s, Age: %d\n", student.Name, student.Age)
    }
}

File: code_translation/exp1_code_translation_security/translated_solutions_claude/Question4/Rust/22.rs
struct Students {
    // internal data structure
}

impl Students {
    fn add(&mut self, name: &str, age: i32) {
        // implementation
    }
}

static mut STUDENTS: Students = Students { /* initialization */ };

fn my_function(name: &str, age: i32) {
    unsafe {
        STUDENTS.add(name, age);
    }
}